package/MDAnalysis/converters/OpenMMParser.py

```diff
@@ -105,16 +108,26 @@ def _mda_topology_from_omm_topology(self, omm_topology):
         -------
         top : MDAnalysis.core.topology.Topology
 
-        """
 
+        .. versionchanged:: 2.2.0
+           * The parser now works when element attribute is missing in some or
+             all the atoms.
+           * When none of the elements are present their atomtypes are guessed
+             using their names and their masses are then guessed using their
+             atomtypes.
+           * When partial elements are present, values from available elements
+             are used whereas the absent elements are assigned an empty string
+             with their atomtype set to 'X' and mass set to 0.0.
+           * For abnormal elements (i.e not in SYM2Z) their symbol is
+             used as it is for atomtypes but an empty string is used for
+             elements.
```

Comment:
Instead of `SYM2Z`, could you please just clarify that the symbols are not real, valid elements?